packages/api/endpoints/distribution.js

```diff
@@ -91,7 +92,12 @@ async function handleRedirectRequest(req, res) {
       {
         expires: new Date(getAccessTokenResponse.expirationTime),
         httpOnly: true,
-        secure: true
+        // Running API locally will be on http, not https, so cookies
+        // should not be set to secure for local runs of the API.
+        //
+        // isLocalApi() is local: true !== true = false (not secure)
+        // isLocalApi() is not local: false !== true = true (secure)
+        secure: (isLocalApi() !== true)
```

Comment:
This could probably be clearer with a helper function.
`secure: cookiesAreSecure()` . 

and `cookisAreSecure = ()  => { if (isLocalApi()) return false; return true}` . Or something like that.  This just seems like a lot of comments to explain that test.

Reply:
Yes, that is much easier to follow